concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Luangthongkum-2019-341-1	1	all, all gone, whole		
+Luangthongkum-2019-341-2	2	ant	587	ANT
+Luangthongkum-2019-341-3	3	arrive	1387	ARRIVE
+Luangthongkum-2019-341-4	4	arrow, dart	977	ARROW
+Luangthongkum-2019-341-5	5	ascend, go up, rise	3207	GO UP OR RISE (ONESELF)
+Luangthongkum-2019-341-6	6	ashes, fireplace		
+Luangthongkum-2019-341-7	7	astringent		
+Luangthongkum-2019-341-8	8	bamboo	1927	BAMBOO
+Luangthongkum-2019-341-9	9	bamboo rat		
+Luangthongkum-2019-341-10	10	bamboo strip		
+Luangthongkum-2019-341-11	11	bamboo shoot		
+Luangthongkum-2019-341-12	12	bark (n.), leather, skin	2127	BARK OR SKIN
+Luangthongkum-2019-341-13	13	bat	1793	BAT
+Luangthongkum-2019-341-14	14	bathe	138	BATHE
+Luangthongkum-2019-341-15	15	bear (animal)	1820	BEAR
+Luangthongkum-2019-341-16	16	bedbug	1949	BEDBUG
+Luangthongkum-2019-341-17	17	bee (Apis cerana)	665	BEE
+Luangthongkum-2019-341-18	18	bee (Apis dorsata)		
```

Comment:
As both these are specific species of honeybees, I would rather leave both unmapped.

Reply:
This was the result of our discussion, see https://github.com/concepticon/concepticon-data/pull/876#discussion_r449245392

Reply:
Ah, I somehow didn't see those comments in the file overview.